_episodes/01-conda.md

```diff
@@ -15,9 +15,10 @@ keypoints:
 
 ## Background
 
-The Python package installer ([pip](https://pip.pypa.io)) only works for libraries written in pure Python.
-Many scientific Python libraries have C and/or Fortran dependencies,
-so the easy solution to this problem is to use a distribution like Anaconda or Canopy, 
+The Python package installer ([pip](https://pip.pypa.io)) works really well for libraries written in pure Python.
```

Comment:
We do have wheels now and saying that `pip` only works for pure python is no longer true.
However, `conda` is still a better choice due to some wheels' limitations and package availability.

(I don't want to get into the details about conda packages vs wheels though.)

Reply:
Thanks, @ocefpaf. You make a good point. Like you say, it doesn't seem like a good idea to go into the details of conda packages vs wheels, so one solution might just be to say the following?

"Until recently, the Python package installer (pip) only worked for libraries written in pure Python. This was a major limitation for the data science community, because many scientific Python libraries have C and/or Fortran dependencies. To spare people the pain of installing these dependencies, distributions like [Anaconda](https://www.anaconda.com/distribution/) and [Canopy](https://www.enthought.com/product/canopy/) have been released, which come with the most popular data science libraries and their dependencies pre-installed. These distributions also come with a package manager for installing libraries that weren’t pre-installed. This tutorial focuses on [conda](https://conda.io/docs/), which is the package manager associated with Anaconda (as we’ll see, it has some advantages over the Canopy package manager)."

Reply:
Much better this way!